app/scrapers/saarland_scraper.rb

```diff
@@ -0,0 +1,181 @@
+require 'date'
+
+module SaarlandScraper
+  BASE_URL = 'http://www.landtag-saar.de'
+
+
+  class Detail < DetailScraper
+    def scrape
+      text = SaarlandScraper.build_text_parameter(@legislative_term, @reference)
+      parameters = "#{text}&r=saarlandcontenttype%3D%22drucksache%22%20saarlandwahlperiode%3D%22#{@legislative_term}%22"
+      mp = mechanize.get "#{BASE_URL}/Service/Seiten/Suche.aspx?#{parameters}"
+      entry = SaarlandScraper.extract_search_entry(mp, @legislative_term, @reference)
+      SaarlandScraper.extract_paper_from_search_entry(entry, @legislative_term, @reference)
+    end
+  end
+
+  class Overview < Scraper
+    SEARCH_URL = BASE_URL + "/Dokumente/Seiten/Drucksachen.aspx?FilterField1=Wahlperiode&FilterValue1="
+
+    def supports_pagination?
+      true
+    end
+
+    def supports_streaming?
+      true
+    end
+
+    def scrape(&block)
+      page = 1
+      papers = []
+      block = -> (paper) { papers << paper } unless block_given?
+      loop do
+        mp = search(page)
+        puts mp.inspect
+        logger.debug "[scrape] page: #{page}"
+        scrape_page(mp, &block)
+        break unless mp.search('//*[@id="bottomPagingCellWPQ2"]/table/tbody/tr/td[3]').size > 0
+        page += 1
+      end
+      papers unless block_given?
+    end
+
+    def scrape_paginated(page)
+      papers = []
+      block = -> (paper) { papers << paper } unless block_given?
+      mp = search(page)
+      logger.debug "[scrape_paginated] page: #{page}"
+      scrape_page(mp, &block)
+      papers unless block_given?
+    end
+
+    def scrape_page(mp, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.